### PR TITLE
[13.0][IMP] base_business_document_import partner search

### PR DESCRIPTION
**Pull request by TDu**
As there is no longer a specific separation between customer and
supplier with the `customer_rank` and `supplier_rank` fields on partner.
It is less error prone to order the partner found by there rank type
insteed of filtering.

### CODE REVIEW

File: base_business_document_import/models/business_document_import.py [hunk 1]
```diff
@@ -71,12 +71,13 @@ def _match_partner(self, partner_dict, chatter_msg, partner_type="supplier"):
...
         if partner_type == "supplier":
-            domain += [("supplier_rank", ">", 0)]
+            order = "supplier_rank desc"
             partner_type_label = _("supplier")
         elif partner_type == "customer":
-            domain += [("customer_rank", ">", 0)]
+            order = "customer_rank desc"
             partner_type_label = _("customer")
         else:
+            order = ""
```
Comment (astirpe) [inline base_business_document_import/models/business_document_import.py:80]: Is `order = ""` working fine? Not crashing? :thinking:

Comment (TDu) [reply base_business_document_import/models/business_document_import.py:80]: I just tested it in a shell and no crash ( I mean the search with order ="")

**Review by astirpe — APPROVED** [review]
LGTM

**Review by jbaudoux — APPROVED** [review]
Thanks

**Pull request merged**